`a11y-high-contrast-light` errors showing up in Sphinx on `main`

I am trying to get some improvements into the `sphinx-book-theme`  using `main` from this theme, and finding this error message that keeps popping up:

```
E         + WARNING: Color theme a11y-high-contrast-light not found by pygments, falling back to tango.
E         + WARNING: Color theme a11y-high-contrast-dark not found by pygments, falling back to monokai.
```

Any idea why this would be happening? cc @drammock who authored the PR where I believe these were added:

- https://github.com/pydata/pydata-sphinx-theme/pull/1064

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,8 @@ dependencies = [
   "beautifulsoup4",
   "docutils!=0.17.0",
   "packaging",
-  "pygments>=2.7"
+  "pygments>=2.7",
+  "accessible-pygments"
 ]
 
 license = { file = "LICENSE" }
@@ -51,7 +52,6 @@ doc = [
   "rich",
   "sphinxext-rediraffe",
   "sphinx-sitemap",
-  "accessible-pygments",
   # For examples section
   "ablog",
   "jupyter_sphinx",

